openapi/components/schemas/QuickBooksOnlineExternalServiceSettings.yaml

```
@@ -0,0 +1,123 @@
+
+type: object
+description: QuickBooks Online settings.
+properties:
+  customer:
+    type: object
+    title: QuickBooks Online customer external service settings
+    properties:
+      sync:
+        type: string
+        description: Specifies how to sync customers.
```

nitpick: I think it would be good to give more detail
```suggestion
        description: Specifies how to synchronize customer data between your Rebilly and QuickBooks Online account.
```